doc/rpki.texi

```diff
@@ -1,5 +1,5 @@
 @c -*-texinfo-*-
-@c This is part of the Quagga Manual.
+@c This is part of the FRRouting Manual.
```

Comment:
in `bgpd.texi` this line is `@c This is part of the Frr Manual.` and in `babeld.texi` its `@c This is part of the FRR Manual.`, so inconsistent as it is, I would go for the latter version with `FRR Manual`, or not?